network-api/networkapi/wagtailpages/pagemodels/base.py

```diff
@@ -742,6 +776,18 @@ class Meta(TranslatableMixin.Meta, WagtailOrderable.Meta):
 
 
 class Homepage(FoundationMetadataPageMixin, Page):
```

Comment:
@danielfmiranda while creating the A/B test I realized that everytime I wanted to edit the `Homepage ` I was able only to access the `A/B testing activity log` and not any other Homepage edit until I paused or ended the A/B test, just wondering if this a normal behaviour, I missed something, or is a thing to consider.

Reply:
Hi @dlopezvsr!

Thanks for bringing this up, that is indeed the expected behavior when running an AB test 👍 users are not able to update or edit a page with an active A/B test running. They have to end the AB test to be able to make changes again.